Source/Diffusion.cpp

```diff
@@ -239,6 +239,61 @@ PeleC::getMOLSrcTerm(
         }
       }
 
+      if (do_isothermal_walls) {
+        // Compute extensive diffusion flux at domain boundaries
+        BL_PROFILE("PeleC::isothermal_wall_fluxes()");
+        for (int dir = 0; dir < AMREX_SPACEDIM; dir++) {
+          if (
+            (typ == amrex::FabType::singlevalued) ||
+            (typ == amrex::FabType::regular)) {
+            if (domlo_isothermal_temp[dir] > 0.0) {
```

Comment:
So if this is > 0, it will trigger the isothermal wall. But in the documentation, you say it will trigger on negative values. I don't see anyone setting 0K walls so maybe just tweak the docs a bit?

Reply:
Will tweak the Docs.